crates/pink-libs/subrpc/src/lib.rs

```diff
@@ -13,6 +13,7 @@ use pink_extension::chain_extension::{signing, SigType};
 use pink_json as json;
 use scale::{Compact, Decode, Encode};
 
+mod hasher;
```

Comment:
Should we make the Hashers pub? Since the prefix calculators within `pub mod storage` is public. If not, users of the prefix calculators out of this crate would have no Hasher to use.

Reply:
Agree, let me change then

Reply:
Done